HUD/SetFloatingHelpTextStyle.md

```diff
@@ -6,14 +6,48 @@ aliases: ["0x788E7FD431BD67F1"]
 
 ```c
 // 0x788E7FD431BD67F1 0x97852A82
-void SET_FLOATING_HELP_TEXT_STYLE(int hudIndex, int p1, int p2, int p3, int p4, int p5);
+void SET_FLOATING_HELP_TEXT_STYLE(int hudIndex, int style, int hudColor, int p3, int arrowPosition, int boxOffset);
 ```
 
 ## Parameters
-* **hudIndex**: 
-* **p1**: 
-* **p2**: 
-* **p3**: 
-* **p4**: 
-* **p5**: 
+* **hudIndex**: The hud index for the floating help message
+* **style**: Value 0 won't show an arrow at all. Values 1, 2 and -2 will display an arrow.
+* **hudColor**: https://docs.fivem.net/docs/game-references/hud-colors/
+* **p3**: No use case. Always 191 in R* scripts
```

Comment:
This is alpha, it ranges from 0 to 255. Anything greater will simply ignore the alpha value.